js/views/components/VerifiedMod.js

```diff
@@ -1,29 +1,47 @@
-import BaseVw from '../baseVw';
-import loadTemplate from '../../utils/loadTemplate';
-import VerifiedMod from '../../models/VerifiedMod';
 import app from '../../app';
+import VerifiedMod from '../../models/VerifiedMod';
+import loadTemplate from '../../utils/loadTemplate';
+import { isHiRez } from '../../utils/responsive';
 import { handleLinks } from '../../utils/dom';
+import BaseVw from '../baseVw';
 
 export default class extends BaseVw {
   constructor(options = {}) {
     const opts = {
-      arrowClass: '',
-      type: {},
-      data: {},
       ...options,
+      initialState: {
```

Comment:
I refactored this guy so you are explicitly sending it the text and classes you want. It gives you a big amount of control as new use cases present themselves. There are helper function below which let you create options for use-cases that are similar or the same.